lib/u3d/commands_generator.rb

```diff
@@ -139,10 +140,12 @@ def run
         c.option '--[no-]download', 'Perform or not downloading before installation. Downloads by default'
         c.option '--[no-]install', 'Perform or not installation after downloading. Installs by default'
         c.option '-p', '--packages PACKAGES', Array, 'Specifies which packages to download/install. Overriden by --all'
+        c.option '-o', '--operating_system STRING', String, "Downloads packages for specific OS [#{oses.join(', ')}]"
```

Comment:
We could say here as well that this is a precision for --no-install. Something like `Requires the --no-install option`